packages/terra-status-view/src/StatusView.module.scss

```diff
@@ -20,25 +15,27 @@
 
   .is-aligned-top {
     align-content: start;
-    grid-template-rows: 0 auto 0;
+    
+    .top-space {
+      flex: 0 0 0;
+    }
   }
 
   .top-space {
-    grid-area: top-space;
+    flex: 1 0 0;
   }
 
   .bottom-space {
-    grid-area: bottom-space;
+    flex: 2 0 0;
   }
 
   .inner-view {
     align-items: center;
     display: flex;
     flex-direction: column;
-    grid-area: content;
     max-height: 100%;
     max-width: 100%;
-    overflow: auto;
+    overflow: unset;
```

Comment:
Oops, we actually want to keep overflow within the inner-view. Let's revert this change. Here's some discussion on this https://github.com/cerner/terra-core/issues/2737